simple/entities/constants.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+# Defaults.
+DEFAULT_DATA_DIR = ".data"
+DEFAULT_INPUT_DIR = os.path.join(DEFAULT_DATA_DIR, "input")
+DEFAULT_OUTPUT_DIR = os.path.join(DEFAULT_DATA_DIR, "output")
+
+OBSERVATIONS_FILE_NAME = "observations.csv"
+
+# Observations CSV columns.
+COLUMN_DCID = "dcid"
+COLUMN_VARIABLE = "variable"
+COLUMN_DATE = "date"
+COLUMN_VALUE = "value"
```

Comment:
nit: if this is a generated CSV, match the column name to the property name, aka:

observationAbout, variableMeasured, observationDate, value

Reply:
I like this idea but will require changes to the mixer code. @shifucun?

Reply:
We do need a bunch of change for consistency; For now, maybe leave as is. And when function is complete, we can do a one-time update?

Reply:
SG. Will add a TODO.

Also, for the entity/dcid column, should we call it `dcid` or leave it as `entity` for now?